pkg/satellite/service.go

```diff
@@ -20,11 +22,12 @@ import (
 type Service struct {
 	Signer
 
-	store DB
+	store  DB
+	logger *zap.SugaredLogger
 }
 
 // NewService returns new instance of Service
-func NewService(signer Signer, store DB) (*Service, error) {
+func NewService(signer Signer, store DB, logger *zap.SugaredLogger) (*Service, error) {
```

Comment:
Move logger as the first argument to keep consistently as the first argument. Also name it as `log`, it shorter and the logging statements are nicer to read.